### DISCUSSION TITLE
Hi  I  still  farming  but  not  received  payments

### DISCUSSION THREAD

**Discussion by valmegar** [Chia-Network/chia-blockchain#15957] [Support]
Hi  I  still  farming  but  not  received  payments

**Comment by BrandtH22** [comment]
Hey, please feel free to jump into the support channel on our discord server (https://discord.gg/chia) for more immediate and thorough support.

We will need to troubleshoot the current state of your farmer to further determine what might be causing the issues of not farming blocks (or not receiving awards for farmed blocks).

- What is your current estimated time to win?
- How long has it been since you last won a reward?
- Are you solo farming or pool farming? (If pooling, are you using OG plots or the official pooling protocol)
- Are there any errors in the log files?

Answering the above will give us a place to start troubleshooting but more information will be needed to identify the issue.
